src/families/cosmos/logic.js

```diff
@@ -196,7 +198,7 @@ export function canUndelegate(account: Account): boolean {
   invariant(cosmosResources, "cosmosResources should exist");
   return (
     cosmosResources.unbondings &&
-    cosmosResources.unbondings.length < COSMOS_MAX_UNBONDINGS
+    cosmosResources.unbondings.length <= COSMOS_MAX_UNBONDINGS
```

Comment:
you are both increasing 6 to 7 and also moving to `<=` which means something was off by 2 ?

Reply:
We can have 7 active unbondings and 7 redelegations it indeed was off by 2 i guess

Reply:
Yes it was my bad i tought it was 5, but I missmatched it with the number of validators

Reply:
i changed also the <= to < after testing i saw my error there sorry :sweat_smile: